src/openrct2-ui/windows/LandRights.cpp

```diff
@@ -271,25 +271,28 @@ static void window_land_rights_invalidate(rct_window* w)
 
 static void window_land_rights_paint(rct_window* w, rct_drawpixelinfo* dpi)
 {
-    int32_t x, y;
+    ScreenCoordsXY screenCoords;
 
-    x = w->windowPos.x + (window_land_rights_widgets[WIDX_PREVIEW].left + window_land_rights_widgets[WIDX_PREVIEW].right) / 2;
-    y = w->windowPos.y + (window_land_rights_widgets[WIDX_PREVIEW].top + window_land_rights_widgets[WIDX_PREVIEW].bottom) / 2;
+    screenCoords = {
```

Comment:
Declare the variable and initialize it in the same statement, like you did before: `auto screenCoords = ScreenCoordsXY{...}`